azkaban-web-server/src/web/js/azkaban/view/job-edit.js

```diff
@@ -260,4 +259,44 @@ $(function () {
   jobEditView = new azkaban.JobEditView({
     el: $('#job-edit-pane')
   });
+
+  formatJobPropertyValues();
+
 });
+
+
+// Improve readability of long job property values
```

Comment:
Can you please add some more details about how the long properties are formatted? The PR description has a very good summary, you can copy text from there :)

The common theme for the function is that instead of multiple properties on 1 line it will split them up so that there is only 1 property per line. A comment mentioning that will provide good context about the function.

Reply:
Done, thanks.